js/index.js

```diff
@@ -1,42 +1,75 @@
-// ITERATION 1
+//Global HTML selectors
+const displayTotal = document.querySelector('#total-value span')
```

Comment:
"displayTotal" is only used in the function "calculateAll". Thus, I would only declare it inside that function. I would also suggest thinking about a more descriptive name of what "displayTotal" actually is. Perhaps "totalValueSpanEl" or something like that.